db/migrate/20160925060545_change_column_name_in_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnNameInPieces < ActiveRecord::Migration
+  def change
+  	rename_column :pieces, :type, :piece_name
```

Comment:
@lasley When i was writing validations for the type column, i was getting an error," Invalid Single table inheritance type". Then i found out that 'type' is a reserved keyword in ActiveRecord. So i changed it. Later i checked @wdehaes 's code for STI branch, i found out he had created sub classes for pieces. Then i was wondering if using type as column was preplanned so it validates using only subclasses. Am i correct? I thought of removing the entire validation and renaming the column back to type, but wanted your reviews before changing it.